code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse.dm

```diff
@@ -13,8 +13,8 @@
 
 	mob_size = MOB_MINISCULE
 	pass_flags = PASSTABLE
-//	can_pull_size = ITEMSIZE_TINY
-//	can_pull_mobs = MOB_PULL_NONE
+	can_pull_size = ITEMSIZE_TINY // Vorestation Edit - Rykka - Uncommented these. Not sure why they were commented out?
+	can_pull_mobs = MOB_PULL_NONE // Vorestation Edit - Rykka - Uncommented these. Not sure why they were commented out?
```

Comment:
These can both be added under mouse_vr.dm and the original commented out bits can stay. Mouse_vr.dm will override this file with any changes that are made.

Reply:
Updooted <3